packages/evolution-frontend/src/actions/LoadingState.ts

```diff
@@ -1,3 +1,10 @@
+/*
+ * Copyright 2024, Polytechnique Montreal and contributors
```

Comment:
Pour les entêtes manquantes sur les fichiers existants, il faut mettre l'année où le fichier a été créé et non 2024. Pour voir cette date, tu peux faire l'historique du fichier avec `git log --pretty=format:"%h%x09%an%x09%ad%x09%s" -- path/to/file` et prendre l'année du plus vieux commit (généralement 2022 pour Evolution)

Reply:
updated